source/android/Renaissance/lib/home_page.dart

```diff
@@ -9,14 +13,238 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
+  var _pageIndex = 0;
+
+  void changePage(int num) {
+    setState(() {
+      _pageIndex = num;
+      print("Deneme");
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: Center(
-        child: Text("homePage"),
-      ),
-      resizeToAvoidBottomInset: false,
-    );
+    var tags = [
+      {
+        "name": "Popular",
+        "tags": [
+          {"name": "Pop1", "effort": 2, "rating": 1.2},
+          {"name": "Pop2", "effort": 2, "rating": 1.3},
+          {"name": "Pop3", "effort": 2, "rating": 1.4},
+          {"name": "Pop4", "effort": 2, "rating": 1.4},
+        ]
+      },
+      {
+        "name": "For You",
+        "tags": [
+          {"name": "For1", "effort": 2, "rating": 1.5},
+          {"name": "For2", "effort": 2, "rating": 1.6},
+          {"name": "For3", "effort": 2, "rating": 1.7},
+        ]
+      },
+      {
+        "name": "New",
+        "tags": [
+          {"name": "New1", "effort": 2, "rating": 1.8},
+          {"name": "New2", "effort": 2, "rating": 1.8},
+          {"name": "New3", "effort": 2, "rating": 1.9},
+        ]
+      },
+    ];
+    var paths = [
+      {"name": "Selam", "effort": 2, "rating": 1.8},
+      {"name": "Muz", "effort": 2, "rating": 1.8},
+      {"name": "Ahoy", "effort": 2, "rating": 1.9},
+      {"name": "Cam", "effort": 2, "rating": 1.5},
+      {"name": "Kar", "effort": 2, "rating": 1.6},
+      {"name": "Araba", "effort": 2, "rating": 1.7},
+      {"name": "Selam", "effort": 2, "rating": 1.8},
+      {"name": "Muz", "effort": 2, "rating": 1.8},
+      {"name": "Ahoy", "effort": 2, "rating": 1.9},
+      {"name": "Cam", "effort": 2, "rating": 1.5},
+      {"name": "Kar", "effort": 2, "rating": 1.6},
+      {"name": "Araba", "effort": 2, "rating": 1.7},
+    ];
+    return MaterialApp(
```

Comment:
You should not return a new app in a page. You should return Scaffold.

Reply:
Thanks, It was a rookie mistake while I was learning  the flutter.